de.metas.swat/de.metas.swat.base/src/main/java/de/metas/invoicecandidate/api/impl/AggregationEngine.java

```diff
@@ -111,16 +116,19 @@ public static AggregationEngine newInstance()
 	private final transient IAggregationFactory aggregationFactory = Services.get(IAggregationFactory.class);
 	private final transient IPriceListDAO priceListDAO = Services.get(IPriceListDAO.class);
 	private final transient IBPartnerDAO bpartnerDAO = Services.get(IBPartnerDAO.class);
+
 	private final transient IDocTypeDAO docTypeDAO = Services.get(IDocTypeDAO.class);
 
 	private static final String ERR_INVOICE_CAND_PRICE_LIST_MISSING_2P = "InvoiceCand_PriceList_Missing";
 
 	//
 	// Parameters
+	private final IBPartnerBL bpartnerBL;
 	private final boolean alwaysUseDefaultHeaderAggregationKeyBuilder;
 	private final LocalDate today;
 	private final LocalDate defaultDateInvoiced;
 	private final LocalDate defaultDateAcct;
+	private final boolean isUpdateLocationAndContactForInvoice;
```

Comment:
like originally written a bit further down, i think this field should be named `updateLocationAndContactForInvoice` (even though the metasfresh- process parent is `IsUpdateLocationAndContactForInvoice` which is fine for me)

Reply:
done